src/libs/antares/study/parameters.cpp

```diff
@@ -177,8 +177,9 @@ const char* StudyModeToCString(StudyMode mode)
         return "Economy";
```

Comment:
A bit off topic, but for any purpose : is **StudyMode** well named ? Or wouldn't it be better to rename it into  **SimulationMode** : after all, we use this mode when we instantiate the simulation, and it's not used in the **Study** (from what I know)

Reply:
Done ! See https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1771/commits/f63d6c7c37bd07c22f8c04559c7eebfd8cc12d79